mlir/test/e2e/CMakeLists.txt

```diff
@@ -40,6 +40,11 @@ if (ROCK_E2E_TEST_ENABLED)
     conv2d_wrw_perf_config
   )
 endif()
+if (ROCMLIR_DRIVER_MISC_E2E_TEST_ENABLED)
```

Comment:
I'd argue this is a good time to get rid of `ROCMLIR_DRIVER_MISC_E2E_TEST_ENABLED`

Reply:
Previously, these tests are only enabled with fixed inputs. I'll add them as regular e2e tests and see if they can pass with random inputs.